### DISCUSSION TITLE
lolMiner 1.53 beta

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#1650] [General]
### Changes 
- Added Eth/Etc/Ubiq + Kaspa dual mining for AMD RX400, RX500, RX5000 and RX 6000 series. Use --dualmode KASPADUAL to request this algorithm.  Requires --dualpool to be set to a Kaspa stratum pool. GPUs that do not support this dual mode will automatically mine Eth/Etc/Ubiq only. This algorithms prefers the newer architectures. Fee is 1%+0% as for all dual mining modes. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Lolliedieb/lolMiner-releases/releases/tag/1.53_beta'>lolMiner 1.53 beta</a>.</em>

**Comment by globalmining-cl** [comment]
When nvidia dual mining?
